proposals/lip-0061.md

```diff
@@ -564,14 +569,20 @@ def verifyAggregateCommit(blockHeader: BlockHeader)-> bool:
     # maxHeightPrecommitted before processing blockHeader and updating the BFT store.
     if aggregateCommit.height > getBFTHeights().maxHeightPrecommitted:
         return False
+    # The heights of aggregate commits must be at least MIN_CERTIFY_HEIGHT.
+    if aggregateCommit.height < MIN_CERTIFICATE_HEIGHT:
+        return False
     # If there are new BFT parameters for a height h, then the chain needs to include
     # an aggregate commit for the block at height h-1. This block and the corresponding
     # certificate authenticate the BFT parameters via the validators hash property.
     # Note that the BFT parameters at height maxHeightCertified+1 are already authenticated
     # by the previous certificate.
-    heightNextBFTParameters = getNextHeightBFTParameters(getBFTHeights().maxHeightCertified+1)
-    if previous function call returns valid height and aggregateCommit.height > heightNextBFTParameters-1:
-        return False
+    try:
+        heightNextBFTParameters = max(getNextHeightBFTParameters(getBFTHeights().maxHeightCertified+1), MIN_CERTIFICATE_HEIGHT + 1)
```

Comment:
Same as above, it should be possible to simplify this

Reply:
I think this would introduce a mistake, see above.